package/libs/wolfssl/patches/101-update-sp_rand_prime-s-preprocessor-gating-to-match.patch

```diff
@@ -0,0 +1,13 @@
+--- a/wolfcrypt/src/sp_int.c
++++ b/wolfcrypt/src/sp_int.c
```

Comment:
I would say that it would be better if in the patch itself, there is included git header, so we know who is the author of the patch, why it is necessary, and if it was upstreamed.

GitHub and GitLab have a quite interesting feature. If you add to the ending of the URL ```.patch``` for commit/pull request like this: https://github.com/wolfSSL/wolfssl/pull/5390.patch , then you can apply it here.

Reply:
This is a great idea, done.